### DISCUSSION TITLE
Text-Vorhersage im HTML-Template

### DISCUSSION THREAD

**Discussion by PrivatePixel** [roe-dl/weewx-DWD#47] [Q&A] [answered]
Hallo,
erst einmal vielen Dank für deine wirklich gelungene Erweiterung. Bei einem Punkt bin ich aber am verzweifeln. Ich würde gerne die Vorhersage als Text einbinden.

Den HTML-Code habe ich schon so angepaßt:
`<div class="col-sm-12" style="margin-bottom:1em">
    #if os.path.exists("dwd/VHDL50_DWEH_LATEST.html")
    #include raw "dwd/VHDL50_DWEH_LATEST.html"
    [Quelle:
    <a
    href="https://www.dwd.de/DE/wetter/vorhersage_aktuell/nordrhein_westfalen/vhs_nrw_node.html"
    target="_blank">DWD</a>]
    #end if
  </div>`

In der weewx.conf sieht es so bei mir aus:

`[DeutscherWetterdienst]
    path = /etc/weewx/skins/Belchertown/dwd
    [[warning]]
        icons = ../dwd/warn_icons_50x50
        states = Nordrhein-Westfalen
        [[[counties]]]
            Stadt Bochum = BOX
        [[[cities]]]
            Stadt Bochum = BOX
            `
Es werden immer nur die Dateien VVHDL50_DWLG_LATEST.html VHDL51_DWLG_LATEST.html,... erzeugt.

Wo liegt hier der Fehler? Es müsste doch eine VHDL50_DWEH_LATEST.html erzeugt werden, oder?

**Comment by roe-dl** [comment, marked as answer]
In `wget-dwd`, Zeile 11, muß der Code des gewünschten Bundeslandes eingetragen werden.

**Reply by PrivatePixel** [reply]
Vielen Dank - hat sofort funktioniert
